### PR TITLE
some CLB cards

### LINKED ISSUE #911: Nimblewright Schematic

### LINKED ISSUE #923: Poison the Blade

### LINKED ISSUE #926: Pseudodragon Familiar

### LINKED ISSUE #928: Reckless Barbarian

### LINKED ISSUE #930: Rescuer Chwinga

### LINKED ISSUE #931: Rug of Smothering

### LINKED ISSUE #932: Saddle of the Cavalier

### LINKED ISSUE #937: Scouting Hawk

### LINKED ISSUE #940: Sharpshooter Elf

### LINKED ISSUE #945: Steadfast Unicorn

### LINKED ISSUE #946: Stoneskin

### LINKED ISSUE #950: Stunning Strike

### LINKED ISSUE #951: Swashbuckler Extraordinaire

### PR DESCRIPTION
**Pull request by kevlahnota**
- closes #945 
- closes #946 
- closes #950 
- closes #951 
- closes #940 
- closes #937 
- closes #932
- closes #931
- closes #930
- closes #928 
- closes #926 
- closes #923 
- closes #911

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Name:Swashbuckler Extraordinaire
+ManaCost:2 R
+Types:Creature Dragon Rogue Warrior
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigTreasure | TriggerDescription$ When CARDNAME enters the battlefield, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigTreasure:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigSac | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, you may sacrifice one or more Treasures. When you do, up to that many target creatures gain double strike until end of turn.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt:7]: "when you do" means ImmediateTrigger

File: forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigSac:DB$ Sacrifice | Defined$ You | Amount$ X | SacValid$ Treasure | SacMessage$ treasure | RememberSacrificed$ True | Optional$ True | SubAbility$ TrigChoose
+SVar:TrigChoose:DB$ ChooseCard | MinAmount$ 0 | Amount$ DBSize | Choices$ Creature | SubAbility$ DBPump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt:9]: Use Targeting

File: forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Name:Swashbuckler Extraordinaire
+ManaCost:2 R
+Types:Creature Dragon Rogue Warrior
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigTreasure | TriggerDescription$ When CARDNAME enters the battlefield, create a Treasure token. (It's an artifact with "{T}, Sacrifice this artifact: Add one mana of any color.")
+SVar:TrigTreasure:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigSac | OptionalDecider$ You | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, you may sacrifice one or more Treasures. When you do, up to that many target creatures gain double strike until end of turn.
+SVar:TrigSac:DB$ Sacrifice | Defined$ You | Amount$ X | SacValid$ Treasure | SacMessage$ treasure | RememberSacrificed$ True | SubAbility$ TrigImmediateTrig
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt:8]: The Optional must still be on the effect I think
else you always have to sac all

Comment (kevlahnota) [reply forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt:8]: should there be two optional? or just move the optional to the sac part

File: forge-gui/res/cardsfolder/upcoming/scouting_hawk.txt [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Name:Scouting Hawk
+ManaCost:2 W
+Types:Creature Bird
+PT:1/1
+K:Flying
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Creature.Self | Execute$ TrigChange | CheckSVar$ Y | SVarCompare$ GTX | TriggerDescription$ When CARDNAME enters the battlefield, if an opponent controls more lands than you, search your library for a basic Plains card, put it onto the battlefield tapped, then shuffle.
+SVar:TrigChange:DB$ ChangeZone | Origin$ Library | Destination$ Battlefield | Tapped$ True | ChangeType$ Plains.Basic | ChangeNum$ 1 | Mandatory$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/scouting_hawk.txt:7]: Hidden & Specific means this isn't mandatory

File: forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+SVar:TrigTreasure:DB$ Token | TokenAmount$ 1 | TokenScript$ c_a_treasure_sac | TokenOwner$ You
+T:Mode$ AttackersDeclared | AttackingPlayer$ You | Execute$ TrigSac | TriggerZones$ Battlefield | TriggerDescription$ Whenever you attack, you may sacrifice one or more Treasures. When you do, up to that many target creatures gain double strike until end of turn.
+SVar:TrigSac:DB$ Sacrifice | Defined$ You | Amount$ X | SacValid$ Treasure | SacMessage$ treasure | Optional$ True | RememberSacrificed$ True | SubAbility$ TrigImmediateTrig
+SVar:TrigImmediateTrig:DB$ ImmediateTrigger | Execute$ DBPump | TriggerDescription$ When you do, up to that many target creatures gain double strike until end of turn.
+SVar:DBPump:DB$ Pump | ValidTgts$ Creature | KW$ Double Strike | TargetMin$ 0 | TargetMax$ DBSize | SubAbility$ DBCleanup | TgtPrompt$ Select target creatures | StackDescription$ {c:Targeted} gain double strike until end of turn. | SpellDescription$ Up to that many target creatures gain double strike until end of turn.
+SVar:DBCleanup:DB$ Cleanup | ClearChosenCard$ True | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt:11]: No longer a reason for ClearChosenCard

File: forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt (hunk 3 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/swashbuckler_extraordinaire.txt]: It could change AI behaviour but two prompts would be annoying I think

**Review by tool4ever — APPROVED** [review]

**Pull request merged**